docs/pages/configuration/pipelines/functions/purge_deployments.mdx

```diff
@@ -2,3 +2,26 @@
 title: Function purge_deployments
 sidebar_label: purge_deployments
 ---
+
+This function removes a deployment.
+
+## Arguments
+
+
+## Usage
+
+```
+purge_deployments deployment1 deployment2
+```
+
+## Flags
+
+By default, flags are disabled. If a flag listed below is enabled, the following action will be taken.
+
+### `--all`
+
+Purges all existing deployments.
+
+### `--sequential`
+
+Purges all existing deployments in sequential order, one after another.
```

Comment:
How is the order determined? By age? By alphabetical order? Would love clarification here.

Reply:
@FabianKramm : This is a great question. Can you help us with this?

Reply:
Thats a good question: the answer is the reverse order of how they were deployed initially. So let's assume you have 2 deployments A and B that are deployed in the order A and then B, then they would be purged in order B and then A. If A and B are deployed in parallel, the `--sequential` purge would essentially be random, so this only makes sense if you are deploying deployments sequentially as well.